tests/integration/rest.integration.test.ts

```diff
@@ -206,7 +209,13 @@ describe('HiFi API REST Calls', () => {
             expect(settingsJSON['name']).toBe(stackData.apps.app1.spaces.space1.name);
         });
 
-        test(`make a space not joinable`, async () => {
+        test(`CAN make a space not joinable`, async () => {
+            try {
+                await fetch(`${stackURL}/api/v1/spaces/${stackData.apps.app1.spaces.space1.id}/settings?token=${adminToken}&new-connections-allowed=true`);
```

Comment:
did you mean to have this in here twice now?

Reply:
I don't think this is a duplicate. I have 4 tests around testing the joinable property
* admin can make a space joinable
* admin can make a space not joinable
* non-admin cannot make a space joinable
* non-admin cannot make a space not joinable
 Before each of these tests I set the property to the opposite of what the attempted change will be. I originally just assumed the property would be in a certain state, but that may not always be true if any test fails before the current one. Maybe we don't need all of these cases?